src/HPCCache/HPCCache/help/Update-AzHpcCache.md

```diff
@@ -0,0 +1,237 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HPCCache.dll-Help.xml
+Module Name: Az.HPCCache
+online version: https://docs.microsoft.com/en-us/powershell/module/az.hpccache/update-azhpccache
+schema: 2.0.0
+---
+
+# Update-AzHpcCache
+
+## SYNOPSIS
+Updates a HPC Cache.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Update-AzHpcCache -ResourceGroupName <String> -Name <String> [-Upgrade] [-Flush] [-Force] [-PassThru] [-AsJob]
```

Comment:
Can `-Upgrade` and `-Flush` be used together? If not, I recommend split them into different parameter sets.

Reply:
Fixed in 0bfa666